apis/apis.json

```diff
@@ -3025,6 +3025,7 @@
       "shortName": "migrationcenter",
       "serviceConfigFile": "migrationcenter_v1.yaml",
       "restNumericEnums": true
```

Comment:
Missing comma, which fails the build.

Reply:
Yup, have repushed. And it looks like we should be able to release fairly soon.